pkgs/by-name/he/healthchecks/package.nix

```
@@ -6,12 +6,14 @@
   python3,
 }:
 let
-  py = python3.override {
+  py = python3.override (previous: {
     self = py;
-    packageOverrides = final: prev: {
-      django = prev.django_5;
-    };
-  };
+    packageOverrides = lib.composeExtensions previous.packageOverrides or (_: _: { }) (
```

I know this doesn't make a difference due to [operator precedence](https://nix.dev/manual/nix/2.24/language/operators), but I think it'd be clearer to read as
```suggestion
    packageOverrides = lib.composeExtensions (previous.packageOverrides or (_: _: { })) (
```